### PR TITLE
Replace hard code of /dev/sdc with common function

### PR DESCRIPTION
**Pull request by xian123**
1. Replace hard code of /dev/sdc with common function
2. Remove the duplicate configuration of STORAGE-VHDX-VERIFY-SECTORSIZE in xml.

### CODE REVIEW

**Comment by xian123** [comment]
Test results:
```
[LISAv2 Test Results Summary]
Test Run On           : 08/23/2019 06:35:22
VHD Under Test        : D:\xhx\vhds\Centos_7.5.vhdx
Initial Kernel Version: 3.10.0-862.el7.x86_64
Final Kernel Version  : 4.14.119-b0de27758059
Total Test Cases      : 3 (3 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:23

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 FCOPY                FCOPY-DISABLE-ENABLE                                                              PASS                 7.71
    2 NETWORK              NET-FILE-COPY-SCP                                                                 PASS                 9.48
    3 STORAGE              STORAGE-VHDX-VERIFY-SECTORSIZE                                                    PASS                 1.79
```
```
[LISAv2 Test Results Summary]
Test Run On           : 08/23/2019 06:36:40
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Initial Kernel Version: 5.0.0-1014-azure
Final Kernel Version  : 5.0.0-1014-azure
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:1:1

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 LTP                  LINUX-TEST-PROJECT-TESTS                                                          PASS                56.08
        proc01 : FAIL
        quota_remount_test01 : FAIL
```

File: Libraries/CommonFunctions.psm1 [hunk 1]
```diff
@@ -1160,18 +1160,21 @@ function Mount-Disk{
...
 	#!/bin/bash
-	# /dev/sdc is used as /dev/sdb is the resource disk by default
-	(echo d;echo;echo w)|fdisk /dev/sdc
-	(echo n;echo p;echo 1;echo;echo;echo w)|fdisk /dev/sdc
+
+	(echo d;echo;echo w)|fdisk $deviceName
+	(echo n;echo p;echo 1;echo;echo;echo w)|fdisk $deviceName
 	if [ $? -ne 0 ];then
 		echo "Failed to create partition..."
 		exit 1
```
Comment (juhlee-microsoft) [inline Libraries/CommonFunctions.psm1:1180]: This is not about this PR, but why are some lines 'exit' and 'return' both? 
'exit 1' in the function modules does not make sense. Can you also fix it?

Comment (xian123) [reply Libraries/CommonFunctions.psm1:1180]: Replace the 'exit' with 'return' and hit the below error.  Keep the 'exit' as it is.
Error: return: can only `return' from a function or sourced script

**Review by juhlee-microsoft — APPROVED** [review]
Thank you for the all testing data

**Pull request merged**